vm/src/obj/rien.rs

```diff
@@ -0,0 +1 @@
+use crate::obj:
```

Comment:
What is the purpose of this file?

Reply:
Oh sorry i didn't see all the mess in my commit. I 'll fix it tomorrow